tests/devices_test/device_packages_test.py

```diff
@@ -18,9 +18,12 @@ def testPackages(self):
         dev = MDRaidArrayDevice("dev", level="raid1", parents=[dev1,dev2])
         luks = LUKSDevice("luks", parents=[dev])
         packages = luks.packages
+        packages.sort() # sort due to blivet/issues/154
 
         # no duplicates in list of packages
-        self.assertListEqual(packages, list(set(packages)))
+        l = list(set(packages))
+        l.sort() # sort due to blivet/issues/154
+        self.assertListEqual(packages, l)
```

Comment:
> We have a list, not set. We convert it to set to remove all duplicates, then back to list to compare it to the original. Now this can be done in other ways as #154 suggests, but there aren't two sets which you can compare to begin with.

We can convert both to sets and compare the sets.